.github/ISSUE_TEMPLATE/az4-preview-bug-report.md

```diff
@@ -0,0 +1,67 @@
+---
+name:  Az 4.x preview bug report 
+about: Report errors or unexpected behaviors specifically for the Az 4.0  module in preview
+title: ''
+labels: 'Az 4.x Preview'
```

Comment:
We need to add this label first, right?

Reply:
Should also have the 'Azure PS Team' label

Reply:
Correct, I have updated the label and created the label.